ci-studio-common-util/cmd/update.go

```diff
@@ -0,0 +1,151 @@
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"runtime"
+
+	"github.com/chef/ci-studio-common/lib"
+	"github.com/juju/fslock"
+	"github.com/mholt/archiver"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	updateCommand = &cobra.Command{
+		Use:   "update",
+		Short: "Update the ci-studio-common install",
+		Run:   maybeUpdateInstall,
+	}
+
+	phase  int
+	force  bool
+	suffix string
```

Comment:
[minor] We found it useful to put opts like this in a struct:

```
var updateOpts = struct{
    phase int
    force bool
    suffix string
}{}
```

Just to help namespace things down the road.

Reply:
That's a great idea! Thanks!